deploy/kubernetes/images/stable-master/image.yaml

```diff
@@ -52,5 +52,5 @@ imageTag:
   # Don't change stable image without changing pdImagePlaceholder in
   # test/k8s-integration/main.go
   newName: k8s.gcr.io/cloud-provider-gcp/gcp-compute-persistent-disk-csi-driver
-  newTag: "v1.8.2"
+  newTag: "v1.9.0"
```

Comment:
I think you need to copy all the rc overlay changes here too.

Reply:
Done

Reply:
I don't see the patches from https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1144 here.

Reply:
https://storage.googleapis.com/kubernetes-jenkins/logs/ci-gcp-compute-persistent-disk-csi-driver-release-staging-k8s-master-integration/1628728505032249344/build-log.txt 

This is not the one to look at? It says csi-attacher v4.2.0

Reply:
Added and build ``bin/kustomize build deploy/kubernetes/overlays/stable-master/``

```
- args:
        - --v=5
        - --csi-address=/csi/csi.sock
        - --http-endpoint=:22012
        - --leader-election
        - --leader-election-namespace=$(PDCSI_NAMESPACE)
        - --timeout=250s
        - --max-grpc-log-length=10000
        - --default-fstype=ext4
        env:
        - name: PDCSI_NAMESPACE
          valueFrom:
            fieldRef:
              fieldPath: metadata.namespace
        image: k8s.gcr.io/sig-storage/csi-attacher:v4.2.0
        livenessProbe:
          failureThreshold: 1
          httpGet:
            path: /healthz/leader-election
            port: http-endpoint
          initialDelaySeconds: 10
          periodSeconds: 20
          timeoutSeconds: 10
        name: csi-attacher
        ports:
        - containerPort: 22012
          name: http-endpoint
          protocol: TCP
        volumeMounts:
        - mountPath: /csi
          name: socket-dir
```